lib/Installation/ModuleRegistration/SeparateRegCodesController.pm

```diff
@@ -42,15 +42,23 @@ sub get_untrusted_GPG_popup {
 sub wait_regcode_page {
     my ($self, $args) = @_;
     YuiRestClient::Wait::wait_until(object => sub {
-            $self->{ModuleRegCodePage}->is_shown({timeout => 0});
+            $self->{SeparateRegCodesPage}->is_shown({timeout => 0});
     }, %$args);
 }
 
-sub add_separate_registration_code {
+sub add_separate_we_registration_code {
     my ($self, $regcode, $timeout) = @_;
     $self->wait_regcode_page({timeout => $timeout, interval => 2,
```

Comment:
let's remove this call in this method and in the next one `add_separate_ha_registration_code`  and call this code from the test itself `tests/installation/module_registration/add_*_regcode.pm`. 
the method that we will call there will be `wait_regcode_page` but we can rename it to "wait_registration_common_regcode_finished" so we avoid to introduce 'page' word in the Page layer.

Actually I was about to merge the PR without asking this, but just in case you want to do it along with the rebase, because there was some CI conflict now, so up to you, just optional :)